perl/t/tracks/build/coerceFeatureType.t

```diff
@@ -24,12 +24,12 @@ my $seq = Seq::Tracks::Build->new(
 );
 
 # unit separator
-my $overlap = "\x1f";
+my $overlap = "/";
 
 my $str =
   '1: Homo sapiens BRCA1/BRCA2-containing complex subunit 3 (BRCC3), transcript variant 2, mRNA. (from RefSeq NM_001018055)';
 my $expected =
-  '1: Homo sapiens BRCA1/BRCA2-containing complex subunit 3 (BRCC3), transcript variant 2, mRNA. (from RefSeq NM_001018055)';
+  '1: Homo sapiens BRCA1,BRCA2-containing complex subunit 3 (BRCC3), transcript variant 2, mRNA. (from RefSeq NM_001018055)';
```

Comment:
Does this example still make grammatical sense? Would it make sense to add a '-' instead of a comma (e.g. This gene encodes a subunit of the BRCA1-BRCA2-containing complex (BRCC)) or is it possible that it wouldn't always make sense to replace with a hyphen?

Reply:
Ok, I can make a ticket for this; we can do this in the YAML config, with a build_field_transformation